src/pages/discover.js

```diff
@@ -102,44 +168,69 @@ const DiscoverPage = ({ router }) => {
-              <Flex width={[1, null, 0.5]} justifyContent="center" alignItems="center" mb={[3, null, 0]}>
-                <P as="label" htmlFor="sort" color="white.full" fontSize="LeadParagraph" pr={2}>
-                  Sort By
-                </P>
-                <select name="sort" id="sort" value={sort} onChange={onChange}>
-                  <option value="most popular">Most Popular</option>
-                  <option value="newest">Newest</option>
-                </select>
-              </Flex>
-
-              <Flex width={[1, null, 0.5]} justifyContent="center" alignItems="center">
-                <P as="label" htmlFor="show" color="white.full" fontSize="LeadParagraph" pr={2}>
-                  Show
-                </P>
-                <select name="show" id="show" value={show} onChange={onChange}>
-                  {tagOptions.map(tag => (
-                    <option key={tag} value={tag}>
-                      {tag}
-                    </option>
-                  ))}
-                </select>
-              </Flex>
+            <Flex justifyContent="space-evenly" flexWrap="wrap" mb={4} width="100%">
+              <NavList as="ul" p={0} m={0} justifyContent="space-around" css="margin-right: 256px;">
+                <Box as="li" px={3}>
+                  <Link href="/discover" passHref scroll={false}>
+                    <StyledLink color="black.600">
+                      <FormattedMessage id="menu.allCollectives" defaultMessage="All Collectives" />
+                    </StyledLink>
+                  </Link>
+                </Box>
+                <Box as="li" px={3}>
+                  <Link href="/discover?offset=0&show=open%20source" passHref scroll={false}>
+                    <StyledLink color="black.600">
+                      <FormattedMessage id="menu.openSourceCollectives" defaultMessage="OpenSource Collectives" />
+                    </StyledLink>
+                  </Link>
+                </Box>
+                <Box as="li" px={3}>
+                  <Link href="/discover?offset=0&show=pledged" passHref scroll={false}>
+                    <StyledLink color="black.600">
+                      <FormattedMessage id="menu.pledgedCollective" defaultMessage="Pledged Collectives" />
+                    </StyledLink>
+                  </Link>
+                </Box>
+                <Box as="li" px={3}>
+                  <Link href="/discover?offset=0&show=other" passHref scroll={false}>
+                    <StyledLink color="black.600">
+                      <FormattedMessage id="menu.others" defaultMessage="Others" />
+                    </StyledLink>
+                  </Link>
+                </Box>
+              </NavList>
+              <NavList as="ul" p={0} m={0} justifyContent="space-around" css="margin: 0;">
+                <Box as="li" px={3}>
+                  <SelectWrapper name="show" id="show" value={show} onChange={onChange}>
```

Comment:
In the initial specification (https://github.com/opencollective/opencollective/issues/1387#issuecomment-469169313) it says:

> use a StyledSelect instead of a regular select

This means using the `StyledSelect` component (`src/components/StyledSelect.js`) that we already have.